command-line-script.sh

```diff
@@ -11,25 +11,42 @@ function die {
 
 INITDIR=$PWD
 
-if [ $# != 2 ]
-then
-  die ${INITDIR} "Needs 2 arguments. Usage: $0 <subject_folder> <name>"
+if [ $# -lt 2 ]; then
+  die ${INITDIR} "Needs 2 arguments and an optional flag. Usage: $0 <subject_folder> <name> <True/False>"
```

Comment:
This is not the right way to pass a flag. Usage should be `$0 [-r] <subject_folder> <name>` where `-r` is used to activate reprozip.